src/main/scala/devices/i2c/I2CPeriphery.scala

```diff
@@ -0,0 +1,34 @@
+// See LICENSE for license details.
+package sifive.blocks.devices.i2c
+
+import Chisel._
+import diplomacy.LazyModule
+import rocketchip.{TopNetwork,TopNetworkModule}
+import uncore.tilelink2.TLFragmenter
+
+trait PeripheryI2C {
+  this: TopNetwork { val i2cConfigs: Seq[I2CConfig] } =>
+  val i2cDevices = i2cConfigs.zipWithIndex.map { case (c, i) =>
+    val i2c = LazyModule(new TLI2C(c))
+    i2c.suggestName(s"i2c$i")
```

Comment:
@terpstra made a fix so you don't have to do this suggest name. Sugest that you call it:

val i2c = i2cConfigs.zipWithIndex ... and then remove the suggestName.